package.json

```diff
@@ -89,7 +89,7 @@
     "edge-currency-bitcoin": "3.7.10",
     "edge-currency-ethereum": "0.11.2",
     "edge-currency-monero": "0.1.2",
-    "edge-exchange-plugins": "git://github.com/EdgeApp/edge-exchange-plugins#22f2110bc",
+    "edge-exchange-plugins": "git://github.com/EdgeApp/edge-exchange-plugins#v0.4.0",
```

Comment:
Please rebase this whole branch on top of develop. This upgrade has already been done there.